pages/data-providers/index.jsx

```diff
@@ -66,29 +70,40 @@ const getHelperMessage = ({ created, duplicated, error }) => {
 
   if (created) {
     return {
-      message: `We found OAI-PMH endpoint at ${created.oaiPmhEndpoint}`,
+      message: (
+        <>
+          We found {created.name} under the provided URL and saved it already.
+          We will start harvesting as soon as we are able to approve the data
+          provider.
+        </>
+      ),
```

Comment:
Thank you for your help. We found {created.name} under the provider URL and added it to our data provider collection. We will start harvesting as soon as we are able to approve it.

Reply:
It's not consistent with the error message when we have it already. I don't feel good without saying 'thank you' but I would rather put it after 'we found' because the goal of the message is to inform, thank you is optional here.

Do you have any idea to rephrase this?